x-pack/plugins/maps/public/classes/styles/vector/properties/dynamic_size_property.js

```diff
@@ -110,6 +110,9 @@ export class DynamicSizeProperty extends DynamicStyleProperty {
 
   _getMbDataDrivenSize({ targetName, minSize, maxSize, minValue, maxValue }) {
     const lookup = this.supportsMbFeatureState() ? 'feature-state' : 'get';
+
+    const stops =
+      minValue === maxValue ? [minValue, minSize] : [minValue, minSize, maxValue, maxSize];
```

Comment:
Can you set to max size? Our documentation states that is the behavior, https://www.elastic.co/guide/en/kibana/7.7/vector-style.html#maps-vector-style-quantitative-data-driven and I believe that was the previous behavior.

<img width="400" alt="Screen Shot 2020-06-05 at 11 33 38 AM" src="https://user-images.githubusercontent.com/373691/83906214-6bf17600-a720-11ea-9a50-8e821ad88708.png">